src/main/java/tsml/classifiers/multivariate/DimensionIndependentEnsemble.java

```diff
@@ -150,7 +150,7 @@ public String toString(){
             dist = modules[m].getClassifier().distributionForInstance(testInstance[m]);
             long predTime = System.currentTimeMillis() - startTime;
             
-            vs.storeModuleTestResult(modules[m], dist, predTime);
+            //vs.storeModuleTestResult(modules[m], dist, predTime);
```

Comment:
This intended? Cant remember the exact ramifications of it, but if was intentional and you've looked into it then that's all good

Reply:
Storing test results internally was mainly a helper thing for cawpe experiments iirc, not needed as generic functionality, so fine if it conflicts with something and isnt needed

Reply:
Was getting a null exception when i was calling it.
Is what i think was happening is that the train was not being created properly. 
Can you review that class, as I think you understand the inner workings of the ensemble classes better.

Reply:
Likely the train results being null here then 
https://github.com/uea-machine-learning/tsml/blob/master/src/main/java/machine_learning/classifiers/ensembles/voting/ModuleVotingScheme.java#L120

Reply:
Ah ok, can look at this in more details tomorrow, but likely can just make DimIndEns extend EnhancedAbstractClassifier to get a trainresults object added on, and measure/store the build time naively as start time first line of buildClassifier, end time last line

May be other implications of making it an EAC too, would need another look

Reply:
Resolved, mainly via deletion of what is now a redundant class, really